caravel/assets/javascripts/SqlLab/components/TableElement.jsx

```diff
@@ -65,11 +67,8 @@ class TableElement extends React.PureComponent {
     this.setState({ sortColumns: !this.state.sortColumns });
   }
 
-  render() {
+  getHeader() {
```

Comment:
could this be pulled out into it's own component called `<TableHeader />`, nice to break things into smaller components, makes things more easily testable.